### PR TITLE
Add OAM Update Delay video test

### PR DESCRIPTION
**Pull request by yuriks**
The hardware is always rendering sprites to the linebuffer 1 line ahead of the current scanline. Therefore, there is a 1-scanline delay between when an OAM update is made and when it's visible. This test demonstrates that by jagging a single 64x64 sprite by changing its position during HBlank at the same time as the BG color.

image

### CODE REVIEW

**Comment by yuriks** [comment]
(I tested this on VC on a n3DS, but I'm pretty confident the result will be the same on a GBA.)

**Comment by yuriks** [comment]
Updated with a new background pattern:

image

File: src/video.c [hunk 1]
```diff
@@ -259,11 +259,139 @@ static void layerToggleExpected(void) {
 	REG_DISPCNT = MODE_0 | BG0_ON;
 }
 
+static char oamDelayInside = 0;
+
+static void oamDelayVcount(void) {
+	while (!(REG_DISPSTAT & LCDC_HBL_FLAG));
+	//REG_DISPCNT ^= BG0_ON;
```
Comment (endrift) [inline src/video.c]: Any reason to leave the commented out code in?

Comment (yuriks) [reply src/video.c]: I don't really remember anymore, but most likely not. I'll remove it.

Comment (endrift) [reply src/video.c]: It doesn't look removed to me.

Comment (yuriks) [reply src/video.c]: Oh, apparently I removed the other instance of this. Oops. Fixed for real now.

**Pull request merged**